src/Views/AuthorView.vala

```diff
@@ -23,7 +23,12 @@ private class AppCenter.AuthorView : Gtk.Box {
             return;
         }
 
-        var author_packages = AppCenterCore.FlatpakBackend.get_default ().get_packages_by_author (package.author, AUTHOR_OTHER_APPS_MAX);
+        Gee.Collection<AppCenterCore.Package> author_packages;
+
+        author_packages = package.author_id == null
+            ? AppCenterCore.FlatpakBackend.get_default ().get_packages_by_author (package.author, AUTHOR_OTHER_APPS_MAX)
+            : AppCenterCore.FlatpakBackend.get_default ().get_packages_by_author_id (package.author_id, AUTHOR_OTHER_APPS_MAX);
+
```

Comment:
I personally prefer using `if` than `foo ? bar : baz` here because the part of `bar` and `baz` is too long in this case. But I don't insist on that.

Instead, if you use `foo ? bar : baz` here you don't need to separate declaration and substitution of `author_packages`:
```suggestion
        var author_packages = package.author_id == null
            ? AppCenterCore.FlatpakBackend.get_default ().get_packages_by_author (package.author, AUTHOR_OTHER_APPS_MAX)
            : AppCenterCore.FlatpakBackend.get_default ().get_packages_by_author_id (package.author_id, AUTHOR_OTHER_APPS_MAX);

```

Reply:
Done!